src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/barcode/BarcodeResource.kt

```diff
@@ -60,10 +61,11 @@ class BarcodeResource(private val barcodeService: BarcodeService, private val us
   )
   fun createBarcode(
     @Parameter(hidden = true) @AuthenticationPrincipal userDetails: UserDetails,
+    @Parameter(hidden = true) @SourceIpAddress sourceIp: String,
```

Comment:
I particularly like the implementation of the `@SourceIpAddress` annotation (see SourceIpAddressMethodArgumentResolver below) 👍

Reply:
Removed class as implementation approach has changed